etc/adminhtml/system.xml

```diff
@@ -36,17 +36,30 @@
                     <label>Enable</label>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                 </field>
+                <field id="cookie" translate="label" type="text" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Cookie Duration</label>
+                    <validate>integer</validate>
+                    <comment>Define Duration of the cookie in mm empty value is equal to session cookies</comment>
+                    <comment> <![CDATA[Define Duration of the cookie <strong>in *days*</strong> empty or 0 is equal to session duration]]></comment>
+                </field>
+                <field id="unselected_countries_text" translate="label" type="select" sortOrder="30" showInDefault="1" showInWebsite="1"
+                       showInStore="1">
+                    <label>Use for Unselected Countries Text</label>
+                    <comment><![CDATA[If "Yes" is selected the default modaltext will be shown for all countries which are not selected <br /> If the detected country is equal the <strong>current store country</strong> no hint will be shown.]]></comment>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                </field>
             </group>
             <group id="popup_values" translate="label" type="text" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1">
                 <label>County Modal Properties</label>
-                <field id="locale" translate="label" type="multiselect" sortOrder="10" showInDefault="1" showInWebsite="1"
+                <field id="countries" translate="label" type="multiselect" sortOrder="10" showInDefault="1" showInWebsite="1"
                        showInStore="1">
                     <label>Countries that should be hinted</label>
-                    <comment>Select countries that should receive the modal hint</comment>
-                    <source_model>Magento\Config\Model\Config\Source\Locale</source_model>
+                    <comment> <![CDATA[Select countries that should receive the modal hint <br /><br /><strong>Notice:</strong> that the country specific fields will be shown after saving the country selection!]]></comment>
+                    <source_model>Magento\Directory\Model\Config\Source\Country</source_model>
                 </field>
                 <field id="editor" translate="label" type="editor" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1">
-                    <label>Modaltext</label>
+                    <label>Modaltext for Unselected Counties</label>
+                    <comment> <![CDATA[<strong>Notice:</strong> this Text will only be shown if the User is using a unknown language beside hinted and default store language and if the field "<strong>Use for Unselected Countries Text</strong>"]]></comment>
```

Comment:
`This text will only be shown if the user is using an unknown language beside hinted and default store language and if the field "<strong>Use for Unselected Countries Text</strong>" is displayed.`